packages/auth-ui/helm/templates/route.yaml

```diff
@@ -0,0 +1,21 @@
+{{- $releaseName := .Release.Name -}}
```

Comment:
we need the route certs here to for NesHarim while in CTS we dont need it since it is implemented in dedicated router